pkg/asset/manifests/additionaltrustbundleconfig.go

```diff
@@ -0,0 +1,126 @@
+package manifests
+
+import (
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+)
+
+var (
+	additionalTrustBundleConfigFileName = filepath.Join(manifestDir, "user-ca-bundle-config.yaml")
+)
+
+const (
+	additionalTrustBundleConfigDataKey = "ca-bundle.crt"
+	additionalTrustBundleConfigMapName = "user-ca-bundle"
+)
+
+// AdditionalTrustBundleConfig generates the additional-trust-bundle-config.yaml files.
+type AdditionalTrustBundleConfig struct {
+	ConfigMap *corev1.ConfigMap
+	File      *asset.File
+}
+
+var _ asset.WritableAsset = (*AdditionalTrustBundleConfig)(nil)
+
+// Name returns a human friendly name for the asset.
+func (*AdditionalTrustBundleConfig) Name() string {
+	return "Additional Trust Bundle Config"
+}
+
+// Dependencies returns all of the dependencies directly needed to generate
+// the asset.
+func (*AdditionalTrustBundleConfig) Dependencies() []asset.Asset {
+	return []asset.Asset{
+		&installconfig.InstallConfig{},
+	}
+}
+
+// Generate generates the CloudProviderConfig.
+func (atbc *AdditionalTrustBundleConfig) Generate(dependencies asset.Parents) error {
+	installConfig := &installconfig.InstallConfig{}
+	dependencies.Get(installConfig)
+
+	if installConfig.Config.AdditionalTrustBundle == "" {
+		return nil
+	}
+	data, err := parseCertificates(installConfig.Config.AdditionalTrustBundle)
+
+	if err != nil {
+		return err
+	}
+
+	cm := &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: corev1.SchemeGroupVersion.String(),
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "openshift-config-managed",
```

Comment:
@abhinavdahiya @jcpowermac according to the proxy workflow [doc](https://docs.google.com/document/d/1y0t0yEOSnKc4abxsjxEQjrFa1AP8iHcGyxlBpqGLO08/edit?usp=sharing) and after talking with @bparees, it appears the installer is creating the configmap in the wrong namespace. The namespace should be `openshift-config` instead of `openshift-config-managed`.

Reply:
No i think it's supposed to be `openshift-config-managed` https://github.com/openshift/api/blob/9ef0612c775d8571388e16d1bc68c4b3df83e7fb/config/v1/types_proxy.go#L49